lib/backend/model/bgp_node.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2017 Tigera, Inc. All rights reserved.
```

Comment:
@heschlie could you review and make sure all the right stuff got copied over here?

Reply:
these types aren't used in the v2 client, so I don't think it matters too much.